samples/scaleout/distributor-upgrade/sample.md

```diff
@@ -70,6 +70,10 @@ In Versions 6 and above there is a new API for enlisting with the distributor. T
 
 snippet:Enlisting
 
+The last parameter, `capacity` is the maximum number of messages in-flight between the distributor and the worker. The worker sends an acknowledgement (ACK) message (also called `ready` message) for each message processed. The distributor keeps track of these ACKs to ensure no worker is flooded with work that it can't process
```

Comment:
@SimonCropp well, it should because it is public but I haven't heard anybody using a custom availability manager.

Re the `capacity` thing I guess ideally it should be documented somewhere in the distributor doco but it was only used in the upgrade guide. How about we keep it where it is for now and create a new issue to add a proper distributor API doco that would include both pieces?